lib/uplinkc/access_scope.go

```diff
@@ -42,6 +103,62 @@ func serialize_scope(scopeHandle C.ScopeRef, cerr **C.char) *C.char {
 	return C.CString(serializedScope)
 }
 
+//export restrict_scope
+// restricts a given scope with the provided caveat and encryptionRestrictions
```

Comment:
needs to look `restrict_scope restricts a given scope with the provided caveat and encryption restrictions`. The function name needs to be at the beginning.

Reply:
Changed the comment